frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/local-volume-set-inner.tsx

```diff
@@ -57,19 +57,19 @@ export const LocalVolumeSetInner: React.FC<LocalVolumeSetInnerProps> = ({
 
   return (
     <>
-      <FormGroup label="Volume Set Name" isRequired fieldId="create-lvs--volume-set-name">
+      <FormGroup label="Volume Set Name" isRequired fieldId="create-lvs-volume-set-name">
         <TextInput
           type={TextInputTypes.text}
-          id="create-lvs--volume-set-name"
+          id="create-lvs-volume-set-name"
```

Comment:
Use `data-test`

Reply:
these `ids` are used as `fieldId` in `FormGroup` component. For eg. at 60 here. And as per this comment (inside formGroup component file),
```/** ID of the included field. It has to be the same for proper working. */
  fieldId: string;
```
it has to be `id` only and not `data-test`. `data-test` can be used, if I am using it just for testing purpose, which is not the case here.